.github/workflows/tests.yml

```diff
@@ -38,11 +40,14 @@ jobs:
                   -DSINGULARITY_USE_HELMHOLTZ=ON \
                   -DSINGULARITY_TEST_HELMHOLTZ=ON \
                   -DSINGULARITY_FORCE_SUBMODULE_MODE=ON \
+                  -DSINGULARITY_USE_V_AND_V_EOS=OFF \
                   -DSINGULARITY_PLUGINS=$(pwd)/../example/plugin \
+                  -DCMAKE_LINKER=ld.gold \
+                  -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Compared to the default build type, which is `RelWithDebInfo`, this drops debug symbols, shrinking the memory footprint during link stage.

Reply:
I'm surprised that's such a huge memory hit to be honest. Honestly though, I find that if I _really_ need debug symbols, I'm compiling with full `Debug` anyway so I'm okay with this change.

Reply:
Yeah. I agree. Is also only for this test. The default build is still "RelWithDebInfo."